examples/tutorials/svelte.md

```diff
@@ -0,0 +1,334 @@
+
+Once created, `cd` into your new project and run the following command to
+install dependencies:
+
+```shell
+deno install
+```
+
+Then, run the following command to serve your new SvelteKit app:
+
+```shell
+deno task dev
+```
+
+Deno will run the `dev` task from the `package.json` file which will start the
+Vite development server. Click the output link to localhost to see your app in
+the browser.
+
+## Configure the formatter
+
+`deno fmt` supports Svelte files with the
+[`--unstable-component`](https://docs.deno.com/runtime/reference/cli/fmt/#formatting-options-unstable-component)
+flag. To use it, run this command:
+
+```sh
+deno fmt --unstable-component
+```
+
+To configure `deno fmt` to always format your Svelte files, add this at the top
+level of your `deno.json` file:
+
+```json
+"unstable": ["fmt-component"]
+```
+
+## Add a backend API
+
+We'll build API routes using SvelteKit's built-in API capabilities. SvelteKit
+allows you to create API endpoints by creating `+server.js` or `+server.ts`
+files in your routes directory.
+
+In the `src/routes` directory, create an `api` folder. In that folder, create a
+`data.json`, which will contain the hard coded dinosaur data.
+
+Copy and paste
+[this json file](https://github.com/denoland/tutorial-with-svelte/blob/main/src/routes/api/data.json)
+into the `src/routes/api/data.json` file. (If you were building a real app, you
+would probably fetch this data from a database or an external API.)
+
+We're going to build out some API routes that return dinosaur information.
+SvelteKit provides a simple way to create API endpoints using server files.
+
+Add the dependencies to your `deno.json` file by updating the imports section:
+
+```json title="deno.json"
+{
+  "imports": {
+    "@oak/oak": "jsr:@oak/oak@^17.1.5",
+    "@tajpouria/cors": "jsr:@tajpouria/cors@^1.2.1"
+  }
+}
+```
+
+Next, create `src/routes/api/dinosaurs/+server.js` to handle the
+`/api/dinosaurs` endpoint. This will return all dinosaurs:
+
+```js title="src/routes/api/dinosaurs/+server.js"
+import { json } from "@sveltejs/kit";
+import data from "../data.json" with { type: "json" };
+
+export function GET() {
+  return json(data);
+}
+```
+
+Then create `src/routes/api/dinosaurs/[id]/+server.ts` to handle individual
+dinosaur requests at `/api/dinosaurs/:id`:
+
+```ts title="src/routes/api/dinosaurs/[id]/+server.ts"
+import { json } from "@sveltejs/kit";
+import type { RequestHandler } from "./$types";
+import data from "../../data.json" with { type: "json" };
+
+export const GET: RequestHandler = ({ params }) => {
+  const dinosaur = data.find((item) => {
+    return item.name.toLowerCase() === params.id.toLowerCase();
+  });
+
+  if (dinosaur) {
+    return json(dinosaur);
+  }
+
+  return json({ error: "Not found" }, { status: 404 });
+};
+```
+
+SvelteKit automatically handles routing based on the file structure. The
+`+server.js` and `+server.ts` files define API endpoints, and the `[id]` folder
```

Comment:
Wondering if we should be consistent and use `+server.ts` everywhere.

Reply:
yes probably, will update!